### ISSUE TITLE
Whole installation directory for Metro 2033 Redux being backed up

### ISSUE THREAD

**Issue by darklinkpower** [mtkennerly/ludusavi#172]
### Ludusavi version

v0.15.2

### Operating system

Windows

### Installation method

Standalone

### Description

Backup for Metro 2033 Redux tries to backup the whole game. I'm not sure why it's doing this because from what I see on PCGW it shouldn't be detecting the installation folder for backup

PCGW data: https://www.pcgamingwiki.com/wiki/Metro_2033_Redux#Game_data

### Logs

![image](https://user-images.githubusercontent.com/1389286/213166454-5d73913c-402e-4726-ab76-846b756931b4.png)

![image](https://user-images.githubusercontent.com/1389286/213167198-34ac7556-282a-4f86-ba96-27afa9fc148b.png)

After killing Ludusavi during backup process

![image](https://user-images.githubusercontent.com/1389286/213167534-121d8546-d3f4-4ec5-989f-3721c79949f0.png)

[config.yaml.txt](https://github.com/mtkennerly/ludusavi/files/10445922/config.yaml.txt)

**Comment by mtkennerly** [comment]
This is because of the Linux path `<path-to-game>/<user-id>/` in the save section on PCGW. The ID is interpreted as `*` (anything). Is it possible to narrow that down to any specific files, like `<path-to-game>/<user-id>/*.sav`?

**Comment by darklinkpower** [comment]
> Is it possible to narrow that down to any specific files, like <path-to-game>/<user-id>/*.sav?

Unfortunately I don't use Linux so I can't check. I remember there being an option to limit saves detection to your system or Windows at least but I can't find it, has that been removed? I remember having another issue like this in the past, maybe even with this game and that solved it.

Also as a suggestion perhaps Ludusavi could introduce safety measures to detect possible errors in paths like this one that results backing up the whole game installation folder? I think it's perfectly safe to assume cases like this are an error

**Comment by mtkennerly** [comment]
> Also as a suggestion perhaps Ludusavi could introduce safety measures to detect possible errors in paths like this one that results backing up the whole game installation folder? I think it's perfectly safe to assume cases like this are an error

Agreed. I've added a filter to ignore `<base>/<storeuserid>`, and I've pushed a [manifest update](https://github.com/mtkennerly/ludusavi-manifest/commit/51d49679dea3e01e176300021d84555763a621a0) to reflect that. This just affected 6 games.

> I remember there being an option to limit saves detection to your system or Windows at least but I can't find it, has that been removed?

Yeah, I removed that option. In practice, it didn't change a lot on Windows (since the overwhelming majority of the data is for Windows already), and it's usually not what you'd want on Linux or Mac, so it could be confusing. You can also get most of the benefit now by toggling specific files/folders.

**Comment by darklinkpower** [comment]
Thanks for the fix!

> Yeah, I removed that option. In practice, it didn't change a lot on Windows (since the overwhelming majority of the data is for Windows already), and it's usually not what you'd want on Linux or Mac, so it could be confusing. You can also get most of the benefit now by toggling specific files/folders.

Regarding this, is there any chance of bringing this back? I think there's always the chance like in this case of getting unintended results due to using paths that are not specific to your system, in my opinion having this behavior as an option that only checks for your system by default would be a better approach.

**Comment by mtkennerly** [comment]
I'm not necessarily opposed to it, but I'd like to have a better idea of how useful it would be. My suspicion would be that it's not really needed as long as there are filters for bad/vague paths like what happened here. If you run into any other cases like this, please let me know so we can identify patterns/frequency.
